sw/host/opentitantool/BUILD

```diff
@@ -26,6 +26,7 @@ rust_binary(
         "src/command/rsa.rs",
         "src/command/set_pll.rs",
         "src/command/spi.rs",
+        "src/command/spxplus.rs",
```

Comment:
nit: should we call this `spx.rs`?

Reply:
If so, we should also rename `sw/host/opentitanlib/src/crypto/spxplus.rs`. If this is a more consistent naming scheme. What are your thoughts on also changing the commands to `opentitantool spx ...` rather than `opentitantool spx-plus ...`?